src/App.js

```diff
@@ -24,13 +29,22 @@ class App extends React.Component {
 
     return (
       <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
-
-        <Carousel />
+        <h1 className="App__title">
+          Carousel with
+          {' '}
+          { images.length }
+          {' '}
+          images
+        </h1>
+        <Carousel
+          images={this.state.images}
```

Comment:
Ты же сделал деструктуризацию
```suggestion
          images={images}
```